presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -107,28 +107,23 @@ public static Response getLogs(String filename, JaxrsParameter<Instant> startDat
         requireNonNull(logLevel);
 
         if (!startDate.isValid() || !endDate.isValid()) {
-            return Response.status(Status.BAD_REQUEST)
-                    .entity("Invalid date format").build();
+            return badRequest("Invalid date format");
```

Comment:
At this point, the expected log format has been lost.

I could add a "get error message" method to `JaxrsParameter`, though, and include relevant information there.